packages/internal-workflow-common/src/activity-options.ts

```diff
@@ -87,6 +80,80 @@ Note that the Temporal Server doesn't detect Worker process failures directly. I
   cancellationType?: coresdk.workflow_commands.ActivityCancellationType;
 }
 
+/**
+ * Options for local activity invocation
+ */
+export interface LocalActivityOptions {
+  /**
+   * Identifier to use for tracking the activity in Workflow history.
+   * The `activityId` can be accessed by the activity function.
+   * Does not need to be unique.
+   *
+   * @default an incremental sequence number
+   */
+  activityId?: string;
+
+  /**
+   * RetryPolicy that define how activity is retried in case of failure. If this is not set, then the server-defined default activity retry policy will be used. To ensure zero retries, set maximum attempts to 1.
```

Comment:
Is this server-defined or core-defined? At least in Go, I think the local activity retry options are defaults set in the SDK.

Reply:
Yeah, server-defined here does not make sense.

Reply:
Hmmm.. probably copied mindlessly, will fix.

Reply:
Changed the comment (not pushed yet) and mentioned that even if max attempts is 1 the LA might be executed more than once due to WFT retries.